src/constants.rs

```diff
@@ -86,3 +87,49 @@ impl Display for ResponseType {
         write!(f, "{self:?}")
     }
 }
+
+#[derive(Debug, Deserialize_repr, Serialize_repr, Copy, Clone)]
+#[repr(u16)]
+pub enum SendRemindersTypes {
+    Disable = 0,
+    Enable = 1,
+}
+
+impl Display for SendRemindersTypes {
+    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+        write!(f, "{:?}", *self as u16)
+    }
+}
+
+#[derive(Debug, Serialize)]
+pub struct InvoiceItem<'i> {
+    pub amount: f64,
+    pub item_name: &'i str,
+}
+
+impl<'i> Display for InvoiceItem<'i> {
+    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+        write!(f, "{self:?}")
```

Comment:
This implementation can be improved

```suggestion
        write!(f, "amount: {}, item_name: {}", self.amount, self.item_name)
```